CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 in development
 --------------
 
+Changed:
```

Comment:
Yeah, actually I might get rid of semicolon. Will push a new commit for that

For the headings (Added/Changed/Fixed/etc), I'm using the format from here: http://keepachangelog.com/en/0.3.0/ - it's not perfect, but at least it's a referenceable style guide